path_config.sh

```diff
@@ -11,19 +11,16 @@
 #   --overrides "do_pretrain = 0"
 
 # Default environment variables for JSALT code. May be overwritten by user.
-# See https://github.com/jsalt18-sentence-repl/jiant for more.
+# See https://github.com/nyu-mll/jiant for more.
 
 ##
 # Example of custom paths for a local installation:
 # export JIANT_PROJECT_PREFIX=/Users/Bowman/Drive/JSALT
 # export JIANT_DATA_DIR=/Users/Bowman/Drive/JSALT/jiant/glue_data
-# export JIANT_DATA_DIR=/home/raghu1991_p_gmail_com/
 # export WORD_EMBS_FILE=~/glove.840B.300d.txt
+# export FASTTEXT_EMBS_FILE=None
```

Comment:
what does `FASTTEXT_EMBS_FILE` do? I think if you want to use precomputed fasttext embs, you set `WORD_EMBS_FILE` to point to it